### DISCUSSION TITLE
I can't change partition style from GPT to MBR or vice versa while updating ventoy

### DISCUSSION THREAD

**Discussion by asmsayem04** [ventoy/Ventoy#1432] [General]
Can't change the partition style to MBR to GPT or vice versa while updating Ventoy. While I need to change the partition style from MBR to GTP or vice versa Ventoy update doesn't work I need to install it again and every time I lose all of my iso files and customization

**Comment by steve6375** [comment]
Why do you need to change style?

**Comment by ventoy** [comment]
Yes. I have noticed in the official document: https://www.ventoy.net/en/doc_start.html
![image](https://user-images.githubusercontent.com/59477474/153814092-a94b15b4-4779-4b20-8488-b652ac0580b6.png)
